src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+                    <td>13</td>
+                    <td>Protobuf Data</td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Type 2 GO Device Data</h4>
+        <p>Sent in response to an IOX Request(0x25) message with a Type Request GO Device Data Message (0x02).</p>
+        <table>
+            <thead>
+                <tr>
+                    <th>Bytes</th>
+                    <th>GO Device Data</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-3</td>
+                    <td>Timestamp (# secs since 2002-01-01)</td>
+                </tr>
+                <tr>
+                    <td>4-7</td>
+                    <td>Latitude (1E-7 deg/bit)</td>
+                </tr>
+                <tr>
+                    <td>8-11</td>
+                    <td>Longitude (1E-7 deg/bit)</td>
+                </tr>
+                <tr>
+                    <td>12</td>
+                    <td>Road Speed (km/hr)</td>
+                </tr>
+                <tr>
+                    <td>13-14</td>
+                    <td>RPM</td>
+                </tr>
+                <tr>
+                    <td>15-18</td>
+                    <td>Odometer (0.1 Km/bit)</td>
+                </tr>
+                <tr>
+                    <td>19</td>
+                    <td>
+                        Status Flags (from LSB):
+                        <br></br>
+                        1st bit: 1 = GPS Valid
+                        <br></br>
+                        2nd bit: 1 = Ignition On
+                        <br></br>
+                        3rd bit: 1 = Engine Bus Activity
+                        <br></br>
+                        4th bit: 1 = Date/Time Valid
+                        <br></br>
+                        5th bit: 1 = Speed From Engine
+                        <br></br>
+                        6th bit: 1 = Odometer From Engine
+                    </td>
+                </tr>
+                <tr>
+                    <td>20-23</td>
+                    <td>Trip Odometer (0.1 Km/bit)</td>
+                </tr>
+                <tr>
+                    <td>24-27</td>
+                    <td>Total Engine Hours (0.1 hours/bit)</td>
+                </tr>
+                <tr>
+                    <td>28-31</td>
+                    <td>Trip Duration (1 second/bit)</td>
+                </tr>
+                <tr>
+                    <td>32-35</td>
+                    <td>Deprecated</td>
+                </tr>
+                <tr>
+                    <td>36-39</td>
+                    <td>Driver ID</td>
+                </tr>
+                <tr>
+                    <td>40-51</td>
+                    <td>GO Device Serial Number</td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Type 3 VIN</h4>
+        <p>Sent in response to an IOX Request(0x25) message with Type Request VIN (0x04).</p>
+        <table>
```

Comment:
we should have tables embedded in a div with the .table-container class until we come up with the final styling so it can be applied later on